awx/main/migrations/_create_system_jobs.py

```diff
@@ -15,9 +16,41 @@
 only if new system job templates were created (i.e. new database).
 '''
 
+def create_collection_jt(apps, schema_editor):	
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')	
+    Schedule = apps.get_model('main', 'Schedule')	
+    ContentType = apps.get_model('contenttypes', 'ContentType')	
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)	
+    now_dt = now()	
+    random_time = now_dt + timedelta(minutes=random.randint(-30,30))	
```

Comment:
This represents a behavioral change - customers are _likely_ to be running this upgrade during business hours, and this is going to move their analytics run from ~midnight to in the middle of the day.  What if we did something like this instead:

```python
now_dt = datetime.datetime.combine(now(), datetime.time.min)`
```